### DISCUSSION TITLE
Dexed new UI

### DISCUSSION THREAD

**Discussion by ghost** [asb2m10/dexed#326] [Ideas]
...

**Comment by asb2m10** [comment]
Hi Martin,

Sorry for the late reply and thanks for the proposition. 

As the "product manager" of Dexed, I think that original look of the brown, close to 8-bit is part of what this project is. By removing that retro/80 look, we are removing Dexed identity and I don't think this is the right path. 

That said the reordering of slider on operator is a great optimization and the vertical operator display is a good idea.

If you are ready to work with this limitation (semi-bitmap graphics, 80 brown hardware feel), I would be happy to see that.

The original UI author (AZ) did a great job to bridge the look with the original synth (DX7) epoch. I think it is important to keep that on the UI level.

Thanks,
Pascal.

**Comment by asb2m10** [comment]
Hi Martin,

Thanks for adapting this.... Yes this is better ! Maybe less 3d depth components would make the thing look cleaner.

That said, I have other projects and if I start working on this, we both have to commit shared time in order to integrate your change into the code. Right now, Dexed priority is to migrate to cmake so arm (raspberry pi) users can built it out of the box. 

I am happy to see UI propositions but I will not work on it within the next six month. Can I ping you once I am fully able to collaborate ?

**Comment by probonopd** [comment]
My 2 cents, visually I prefer the Dexed look as-is but logically the structure of https://imgur.com/a/kF5EKmR seems much easier to understand. Are there still all settings there?

![image](https://user-images.githubusercontent.com/2480569/170882659-2cdf522d-c995-46de-b680-60670d1cf9c1.png)

Maybe the new UI structure could be used without changing the visual look?
